content/doc/book/system-administration/security.adoc

```diff
@@ -27,6 +27,8 @@ Securing Jenkins has two aspects to it.
 
 == Access Control
 
+=== Access control for users
+
```

Comment:
I don't gain much from a heading that is immediately followed by a heading with no intervening content.  Could we just rely on the parent heading, then let the user understand that "Access control for builds" is a sub-heading of "Access Control"?
```suggestion
```

Reply:
I will remove it, just thought it would look "asymmetric" and perhaps odd in the TOC.